cmd/osbuild-upload-gcp/main.go

```diff
@@ -57,59 +57,59 @@ func main() {
 	case "rhel-9":
 		guestOSFeatures = gcp.GuestOsFeaturesRHEL9
 	default:
-		logrus.Fatalf("[GCP] Unknown OS Family %q. Use one of: 'rhel-8', 'rhel-9'.", osFamily)
+		olog.Fatalf("[GCP] Unknown OS Family %q. Use one of: 'rhel-8', 'rhel-9'.", osFamily)
```

Comment:
I wanted to keep things simple but it is super easy to create a prefix support:

```go
olog.WithPrefix("GCP").Fatalf("...")
```

Maybe in a followup, or perhaps it is too much. The simplicity of this API is its best strength I think.

Reply:
That's not a bad idea.  Let's consider it for a follow-up.